### PR TITLE
fix(back-to-top): icon and label positioning

### LINKED ISSUE #1173: Docs > Components > Back to top : fix css icon position and alignment
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In component back to top, there are 2 issues :
- icon position :
  - same problem and resolution as in pagination icons, see issue #1119
  - seems that icon is a bit too low 
image
image

- alignment 
  - Visible at this time on main branch, not on published version
  - Problem is well visible on black background :
image
image

### Reduced test cases

see issue #1119 to fix first icon positionning

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

main branch (v5.2)

### PR DESCRIPTION
**Pull request by hannahiss**
For a better alignment: 

- pull up icon of 1px
- pull up label of 1px

Closes #1173

### CODE REVIEW

File: scss/_back-to-top.scss [hunk 1]
```diff
@@ -39,6 +44,7 @@
 
     &[data-#{$boosted-prefix}label]::before {
       position: absolute;
+      top: $back-to-top-title-offset-top;
```
[the remark this answers is not in the document]
Comment (hannahiss) [reply scss/_back-to-top.scss:47]: The transform is about icon, but here I add a top offset on the title....?

**Review by louismaximepiton — APPROVED** [review]
The changes look good to me, the [label inside](https://deploy-preview-1273--boosted.netlify.app/docs/5.1/components/back-to-top/#label-inside) one seems to be a bit weird to me, but I don't know if this is worth to add a rule for it.
Something like `transform: translateY(1px) rotate(.25turn);` l.39.

**Comment by hannahiss** [comment]
> > The changes look good to me, the [label inside](https://deploy-preview-1273--boosted.netlify.app/docs/5.1/components/back-to-top/#label-inside) one seems to be a bit weird to me, but I don't know if this is worth to add a rule for it. Something like `transform: translateY(1px) rotate(.25turn);` l.39.
> 
> Can't we fix [label inside](https://deploy-preview-1273--boosted.netlify.app/docs/5.1/components/back-to-top/#label-inside)?

If I let down the modification on the icon, and only pull up the label, that seems correct to me in both cases. The slightly low icon is not really visible. Could be a good compromise? Or not... 

Standard case: 
image

Label inside case:
image

**Review by julien-deramond — APPROVED** [review]
OK the compromise allows us to have a better rendering in most cases so let's :rocket:

**Pull request merged**